### PR TITLE
set username when request -> accept

### PR DESCRIPTION
**Pull request by IsaakC**
previously it wasnt set until so username was blank during transition from requested -> accepted
also got rid of some unused xml code
image

### CODE REVIEW

**Review by KateVlaar — APPROVED** [review]

File: app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java [hunk 1]
```diff
@@ -135,6 +135,16 @@ private void redraw()
                 this.btn2.setText(getString(R.string.lend_book));
                 this.btn1.setOnClickListener(this::btnSetLocation);
 
+                TextView user;
+                user = bookDetailView.findViewById(R.id.book_request_user);
+                if (this.selectedBook.getRequesters().size() > 0)
+                {
+                    System.out.println("yes");
```
Comment (huddy987) [inline app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java:142]: Please remove the print

**Review by huddy987 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java (hunk 1 above)
Comment (IsaakC) [reply app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java:142]: whoops good catch

**Review by JawadHossain — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -67,4 +67,5 @@ app/build.gradle
 app/src/main/res/values/api_keys.xml
 app/src/main/res/values/test_accounts.xml
 app/src/main/res/values/secrets.xml
-app/src/main/java/com/example/bookwormadventuresdeluxe2/Utilities/NotificationUtility/CloudMessagingServerKeyConstant.java
\ No newline at end of file
+app/src/main/java/com/example/bookwormadventuresdeluxe2/Utilities/NotificationUtility/CloudMessagingServerKeyConstant.java
+app/src/main/java/com/example/bookwormadventuresdeluxe2/NotificationUtility/CloudMessagingServerKeyConstant.java
```
Comment (huddy987) [inline .gitignore:71]: This should not be added

**Review by huddy987 — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -67,5 +67,4 @@ app/build.gradle
 app/src/main/res/values/api_keys.xml
 app/src/main/res/values/test_accounts.xml
 app/src/main/res/values/secrets.xml
-app/src/main/java/com/example/bookwormadventuresdeluxe2/Utilities/NotificationUtility/CloudMessagingServerKeyConstant.java
```
Comment (huddy987) [inline .gitignore]: Leave this one in the gitignore as it is the new path, remove the one below. Remove the other file from github pls

**Review by huddy987 — CHANGES_REQUESTED** [review]

**Review by huddy987 — APPROVED** [review]

**Pull request merged**